tests/e2e/tests/startupCalls.test.js

```diff
@@ -29,6 +29,8 @@ describe('Calls after logging in', () => {
 
   test('Templates', async () => {
     const responseEnv = await utils.fetchReq('projects?type=template');
+    console.log(responseEnv); // Log the entire response
+    console.log(responseEnv.data); // Log the data specifically
```

Comment:
Is this only meant for debugging, or is it meant to stay?

Reply:
Yes I have forgotten to remove it, thanks for pointing it out.